test/integration/h2_capture_fuzz_test.cc

```diff
@@ -0,0 +1,19 @@
+#include "test/integration/h2_fuzz.h"
+
+namespace Envoy {
+void H2FuzzIntegrationTest::initialize() {
+  setDownstreamProtocol(Http::CodecClient::Type::HTTP2);
+  setUpstreamProtocol(FakeHttpConnection::Type::HTTP2);
+
+  HttpIntegrationTest::initialize();
+}
+
+DEFINE_PROTO_FUZZER(const test::integration::H2CaptureFuzzTestCase& input) {
```

Comment:
Now that my PR landed with support for persistent mode, you can easily add this here to get two fuzzers, one normal and one with persistent mode for speed.

Reply:
Indeed. Was thinking of getting this PR into better shape first, and then to merge master, and implement this.